ports/stm32/dma.c

```diff
@@ -1108,6 +1327,12 @@ void dma_nohal_init(const dma_descr_t *descr, uint32_t config) {
     } else {
         __HAL_DMA2_REMAP(descr->sub_instance);
     }
+    #elif defined(STM32G4)
+    // descr = { DMA1_Channel4, DMA_REQUEST_UART3_TX, dma_id_3,  &dma_init_struct_spi_i2c };
+    // #error Hier moet ik nog ff naar kijken hoe we dit aanpassen aan de G4 serie
+    // in L4 wordt hier channel geselecteerd, in G4 ws via DMAMUX (net als H7?)
```

Comment:
If this comment is warranted, please convert it to English :smile:

Reply:
Removed it, was a comment from the start. DMA is kind of different in this G4 compared to most ST processors, as it has a DMA and DMAMUX peripheral, and I had to learn how ST implemented the DMA.